flaml/ml.py

```diff
@@ -134,15 +134,21 @@ def get_estimator_class(task, estimator_name):
 
 def metric_loss_score(
     metric_name,
-    y_predict,
-    y_true,
-    labels=None,
+    y_processed_predict,
+    y_processed_true,
+    label_list=None,
     sample_weight=None,
     groups=None,
 ):
+    # y_processed_predict and y_processed_true are both converted into labels if the original was the token labels

```

Comment:
converted into "labels" if the original was the "token labels"
What does that mean?